README.md

```diff
@@ -58,6 +58,31 @@ options:
   --kernel PATH         Kernel file to calculate hash from
   --initrd PATH         Initrd file to calculate hash from (use with --kernel)
   --append CMDLINE      Kernel command line to calculate hash from (use with --kernel)
+  --guest-features      Hex representation of the guest kernel features expected to be included (Defaults to 0x21).
+
+                        | BIT FIELD | Description |
+                        |:---------:|:------------|
+                        | 0 | SNPActive |
+                        | 1 | vTOM |
+                        | 2 | ReflectVC |
+                        | 3 | RestrictedInjection |
+                        | 4 | AlternateInjection |
+                        | 5 | DebugSwap |
```

Comment:
I usually put the usage section in this README by pasting the output of

    COLUMNS=100 python3 ./sev-snp-measure.py --help

but if you try that, you get something like:

```
  --initrd PATH         Initrd file to calculate hash from (use with --kernel)
  --append CMDLINE      Kernel command line to calculate hash from (use with --kernel)
  --guest-features VALUE
                        Hex representation of the guest kernel features expected to be included (Defaults to 0x21). | BIT FIELD |
                        Description | |:---------:|:------------| | 0 | SNPActive | | 1 | vTOM | | 2 | ReflectVC | | 3 |
                        RestrictedInjection | | 4 | AlternateInjection | | 5 | DebugSwap | | 6 | PreventHostIBS | | 7 | BTBIsolation
                        | | 8 | VmplSSS | | 9 | SecureTSC | | 10 | VmgexitParameter | | 11 | Reserved, SBZ | | 12 | IbsVirtualization
                        | | 13 | Reserved, SBZ | | 14 | VmsaRegProt | | 15 | SmtProtection | | 63:16 | Reserved, SBZ | Ex. **
                        --guest-features 0x21 -> DebugSwap + SNPActive ** --guest-features 0x211 -> SecureTSC + DebugSwap + SNPActive
  --output-format {hex,base64}
                        Measurement output format
```

not exactly the pretty table we want...

I suggest adding a separate section in the README which includes this table ("SNP guest features field values"). In the argument description have something shorter like (untested):

```python3
 parser.add_argument('--guest-features', metavar='VALUE', type=int,
                         default=0x21,
                        help="Hex value of the guest kernel features expected to be included (defaults to 0x21); see README.md for possible values.")
```

In the new README section we can also description the evolution (until kernel 6.6 the value was 0x1 but then it was changed to 0x21 (see commit WXYZ)).